File: update_leads.js
import fs from 'fs';

// Read the leadpool
const leadpool = JSON.parse(fs.readFileSync('src/data/leadpool.json', 'utf8'));

const updates = [
  {
    id: 'org-699',
    dataQuality: 'rejected',
    notes: ['REJECTED: Basic IT reseller (PC247). No ITAD processing facilities, just laptop/PC buying/selling.'],
    stage: 'rejected',
    score: null,
    isPipeline: false,
    lastVerified: new Date().toISOString()
  },
  {
    id: 'org-700',
    dataQuality: 'rejected', 
    notes: ['REJECTED: No web presence found. Cannot verify ITAD operations.'],
    stage: 'rejected',
    score: null,
    isPipeline: false,
    lastVerified: new Date().toISOString()
  },
  {
    id: 'org-703',
    dataQuality: 'rejected',
    notes: ['REJECTED: Computer systems/software sales company (Basingstoke). Not ITAD processing.'],
    stage: 'rejected',
    score: null,
    isPipeline: false,
    lastVerified: new Date().toISOString()
  },
  {
    id: 'org-705',
    dataQuality: 'rejected',
    notes: ['REJECTED: Consumer electronics retailer (Narmi-tech Shopify store). Sells refurbished Apple products to consumers, not B2B ITAD.'],
    stage: 'rejected', 
    score: null,
    isPipeline: false,
    lastVerified: new Date().toISOString()
  },
  {
    id: 'org-709',
    dataQuality: 'rejected',
    notes: ['REJECTED: IBM hardware reseller/refurbisher (neu-comp). Limited to IBM equipment resale, not comprehensive ITAD services.'],
    stage: 'rejected',
    score: null,
    isPipeline: false,
    lastVerified: new Date().toISOString()
  },
  {
    id: 'org-711',
    dataQuality: 'rejected',
    notes: ['REJECTED: No verifiable web presence found for Newelltek Wellington Somerset. Cannot confirm ITAD operations.'],
    stage: 'rejected',
    score: null,
    isPipeline: false,
    lastVerified: new Date().toISOString()
  },
  {
    id: 'org-748',
    dataQuality: 'verified',
    notes: ['QUALIFIED: Resource Development UK - BSI-certified remanufacturer (BS Standards 8887-220/211). Est. 1994, WEEE accredited, 100% UK facility in Rochester. Serves education/business/public sector.'],
    stage: 'qualified',
    score: 75,
    isPipeline: true,
    services: ['Computer Remanufacturing', 'Refurbishment', 'BSI-Certified Process'],
    certifications: ['BSI BS 8887-220', 'BSI BS 8887-211', 'WEEE Exemption'],
    lastVerified: new Date().toISOString()
  },
  {
    id: 'org-752',
    dataQuality: 'rejected',
    notes: ['REJECTED: Recommerce IT - refurbished equipment retailer with consumer focus. No evidence of corporate ITAD services or data destruction facilities.'],
    stage: 'rejected',
    score: null,
    isPipeline: false,
    lastVerified: new Date().toISOString()
  }
];

// Apply updates
updates.forEach(update => {
  const index = leadpool.findIndex(lead => lead.id === update.id);
  if (index !== -1) {
    Object.assign(leadpool[index], update);
    leadpool[index].updatedAt = new Date().toISOString();
    console.log(`Updated ${update.id}: ${leadpool[index].name} - ${update.dataQuality}`);
  }
});

// Write back to file
fs.writeFileSync('src/data/leadpool.json', JSON.stringify(leadpool, null, 2));
console.log('Updates complete');

File: update_leads.py
#!/usr/bin/env python3

import json
from datetime import datetime

def update_leadpool():
    # Load current data
    with open('src/data/leadpool.json', 'r') as f:
        data = json.load(f)
    
    # Define updates
    updates = {
        "org-554": {  # AYOO SERVICES
            "dataQuality": "verified",
            "stage": "qualified", 
            "score": 74,
            "isPipeline": True,
            "notes": ["AYOO Services - Remanufacturing laptops with own facility. ITAD services: secure data erasure (NIST/DoD/GDPR), asset tracking, certified destruction, chain of custody."],
            "services": ["ITAD", "Data Erasure", "Remanufacturing", "IT Recycling"],
            "lastVerified": datetime.utcnow().isoformat() + "Z",
            "updatedAt": datetime.utcnow().isoformat() + "Z"
        },
        "org-588": {  # Comtek Network Systems
            "dataQuality": "verified",
            "stage": "qualified",
            "score": 70, 
            "isPipeline": True,
            "notes": ["Comtek Network Systems - Telecom/network equipment repair, refurb, and recycling. Component-level repairs, circular economy focus. Multi-vendor support."],
            "services": ["Network Equipment Repair", "Refurbishment", "Component Recovery", "Recycling"],
            "lastVerified": datetime.utcnow().isoformat() + "Z",
            "updatedAt": datetime.utcnow().isoformat() + "Z"
        },
        "org-546": {  # Air space technology
            "dataQuality": "rejected",
            "stage": "rejected",
            "score": None,
            "isPipeline": False,
            "notes": ["REJECTED: Apple device reseller/refurbisher only. Not ITAD processor with own facilities."],
            "services": [],
            "lastVerified": datetime.utcnow().isoformat() + "Z",
            "updatedAt": datetime.utcnow().isoformat() + "Z"
        },
        "org-549": {  # APPLEBITE2NDBITE
            "dataQuality": "rejected",
            "stage": "rejected", 
            "score": None,
            "isPipeline": False,
            "notes": ["REJECTED: Apple device retailer only. No ITAD processing facilities."],
            "services": [],
            "lastVerified": datetime.utcnow().isoformat() + "Z",
            "updatedAt": datetime.utcnow().isoformat() + "Z"
        },
        "org-594": {  # DISCOVERY COMPUTER SERVICES
            "dataQuality": "rejected",
            "stage": "rejected",
            "score": None,
            "isPipeline": False, 
            "notes": ["REJECTED: IT equipment buyback service only. No ITAD processing or data destruction facilities."],
            "services": [],
            "lastVerified": datetime.utcnow().isoformat() + "Z",
            "updatedAt": datetime.utcnow().isoformat() + "Z"
        },
        "org-638": {  # INFOCUS ID
            "dataQuality": "rejected", 
            "stage": "rejected",
            "score": None,
            "isPipeline": False,
            "notes": ["REJECTED: ID card printer supplier only. Not ITAD."],
            "services": [],
            "lastVerified": datetime.utcnow().isoformat() + "Z",
            "updatedAt": datetime.utcnow().isoformat() + "Z"
        }
    }
    
    # Apply updates
    for item in data:
        if item["id"] in updates:
            update_data = updates[item["id"]]
            for key, value in update_data.items():
                item[key] = value
            print(f"Updated {item['id']}: {item['name']} -> {item['dataQuality']}")
    
    # Save updated data
    with open('src/data/leadpool.json', 'w') as f:
        json.dump(data, f, indent=2)
    
    print("Leadpool updated successfully")

if __name__ == "__main__":
    update_leadpool()